test/parallel/test-benchmark-url.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+require('../common');
+
+const runBenchmark = require('../common/benchmark');
+
+runBenchmark('url', ['n=1'], { NODEJS_BENCHMARK_ZERO_ALLOWED: 1 });
```

Comment:
Please also add options for all existing options in any url benchmark that exists and pass in a concrete value. Otherwise it would test for all options and that increases the runtime significantly.

So e.g. `method`, `type`...

Reply:
@BridgeAR when I add option like 'method=get' or 'method=' it throws an error:

```
/Users/daynin/Documents/projects/node/benchmark/url/legacy-vs-whatwg-url-get-prop.js:89
      throw new Error(`Unknown method "${method}"`);
      ^

Error: Unknown method "get"
```

or

```
/Users/daynin/Documents/projects/node/benchmark/url/legacy-vs-whatwg-url-get-prop.js:89
      throw new Error(`Unknown method "${method}"`);
      ^

Error: Unknown method ""
```

Can I specify certain benchmark for the test somehow or avoid these errors?

Reply:
Seems like you're passing in `method="get"` given the quotation marks in the error? Instead pass in `method=get`.

Reply:
@apapirovski I passed 'method=get' as an option. But I think the problem is `url-searchparams-read.js` has than config:

```
const bench = common.createBenchmark(main, {
  method: ['get', 'getAll', 'has'],
  param: ['one', 'two', 'three', 'nonexistent'],
  n: [2e7]
})
```

but `legacy-vs-whatwg-url-get-prop.js` has that config:

```
const bench = common.createBenchmark(main, {
  type: Object.keys(inputs),
  method: ['legacy', 'whatwg'],
  n: [1e5]
})
```

There are two params `method` with different values

Reply:
Ah, sorry, didn't see the file name you mentioned. I would probably just rename the `method` to be something else in the legacy benchmark, if that's the only one blocking this.